docs/src/pages/component-status.mdx

```diff
@@ -5,19 +5,28 @@ description: A detailed breakdown of our components and status of their implemen
 
 import { ComponentStatusList } from "../components/ComponentStatus";
 import Layout from "../components/Layout";
+import useMdxPageTableofContents from "../hooks/useMdxPageTableofContents.ts";
```

Comment:
```suggestion
import useMdxPageTableofContents from "../hooks/useMdxPageTableofContents";
```

Reply:
As I noted to Victor in a different PR, the eslint rules you set up expect extensions to be tsx. So when the files are ts, it complains. Should I 1. change the file extension, 2. change the rule, or 3. leave it as is?

Reply:
Sorry about the ESLint mistake 🤦 it should be changed to allow `.ts`